app/assets/stylesheets/_sidebar.scss

```diff
@@ -0,0 +1,70 @@
+.content {
+  position: relative;
+
+  .organization {
+    $actions-height: 3 * $base-spacing;
+
+    &-sidebar {
+      @include span-columns(3 of 12);
+    }
+
+    &-listing {
+      $listing-height: 2 * $base-spacing;
+      position: relative;
+      padding: $base-spacing/2 0;
+
+      &:not(:last-child) {
+        border-bottom: $base-border;
+      }
+
+      &:last-child {
+        margin-bottom: $base-spacing;
+      }
+    }
+
+    &-invalid {
+      &:before {
+        $size: 24px;
+        background: image_url("warning.svg") center no-repeat;
+        content: "";
+        display: block;
+        height: $size;
+        left: -1.5 * $size;
+        position: absolute;
+        width: $size;
+      }
+    }
+
+    &-search {
+      @include fill-parent;
+      height: $actions-height;
+
+      input {
```

Comment:
Empty rule<br>Selector should have depth of applicability no greater than 2, but was 3